scripts/lldb.js

```diff
@@ -60,26 +68,39 @@ function cloneHeaders(lldbVersion, buildDir) {
 
   if (!fs.existsSync(lldbInstallDir)) {
     console.log(`\nCloning lldb ${lldbHeadersBranch} into ${lldbInstallDir}`);
-    // use `git clone --filter` in git v2.19 to only download `lldb` dir of `llvm-project` monorepo
-    // see: https://stackoverflow.com/a/52269934/3117331
-    // see: https://github.blog/2020-01-17-bring-your-monorepo-down-to-size-with-sparse-checkout/
-    child_process.execFileSync(
-        'git', ['clone',
-          '--depth', '1',
-          '--filter=blob:none',
-          '--sparse',
-          '--branch', lldbHeadersBranch,
-          'https://github.com/llvm/llvm-project.git',
-          lldbInstallDir
-        ],
-        { stdio: 'inherit' });  // show progress
-    child_process.execFileSync(
-        'git', [
-          '-C', lldbInstallDir,
-          'sparse-checkout',
-          'set', 'lldb'
-        ],
-        { stdio: 'inherit' });  // show progress
+    if (doesGitSupportSparseCheckout()) {
+      // use `git clone --filter` in git v2.19 to only download `lldb` dir of `llvm-project` monorepo
+      // see: https://stackoverflow.com/a/52269934/3117331
+      // see: https://github.blog/2020-01-17-bring-your-monorepo-down-to-size-with-sparse-checkout/
+      child_process.execFileSync(
+          'git', ['clone',
+            '--depth', '1',
+            '--filter=blob:none',
+            '--sparse',
+            '--branch', lldbHeadersBranch,
+            'https://github.com/llvm/llvm-project.git',
```

Comment:
Maybe could we put that one into a variable ?

Reply:
Hey @tony-go I'm looking to merge this.
Can you be a bit more specific with the point about the variable? 
I'm not 100% on what your ask is.

Reply:
I think the ask is to put the Github URL into a variable. I don't feel particularly strongly about it; feel free to do it before merging if you'd like.

Reply:
OK this has been around for a while with no updates. 
@tony-go I'm also not strongly pushed about changing the implementation.
I'll merge this and please feel free to PR your suggested change if you want.